include/mbedtls/pk.h

```diff
@@ -361,24 +361,35 @@ int mbedtls_pk_setup(mbedtls_pk_context *ctx, const mbedtls_pk_info_t *info);
 /**
  * \brief           Initialize a PK context to wrap a PSA key.
  *
- * \note            This function replaces mbedtls_pk_setup() for contexts
- *                  that wrap a (possibly opaque) PSA key instead of
- *                  storing and manipulating the key material directly.
+ *                  This function helps creating a PK context which wraps a
+ *                  PSA key. The PSA wrapped key must:
+ *                  * remain valid as long as the wrapping PK context is in use,
+ *                    that is at least between the point this function is
+ *                    called and the point mbedtls_pk_free() is called on this
+ *                    context;
+ *                  * be a key pair;
+ *                  * be an EC or RSA type (DH is not suported in PK module).
+ *
+ *                  Under the hood PSA functions are used to perform the required
+ *                  operations and, based on the key type, used algorithms will be:
+ *                  * EC:
+ *                      * verify, verify_ext: #PSA_ALG_ECDSA_ANY;
+ *                      * sign, sign_ext: try #PSA_ALG_DETERMINISTIC_ECDSA()
+ *                        first and, in case it fails, try with #PSA_ALG_ECDSA().
+ *                  * RSA:
+ *                      * sign, sign_ext: use the algorithm associated with the
+ *                        wrapped PSA key;
+ *                      * verify: not supported;
+ *                      * verify_ext: not supported;
+ *                      * decrypt: #PSA_ALG_RSA_PKCS1V15_CRYPT;
+ *                      * encrypt: not supported.
+ *                  In order for the above operations to succeed, the policy of
+ *                  the wrapped PSA key must allow the specified algorithm.
  *
  * \param ctx       The context to initialize. It must be empty (type NONE).
  * \param key       The PSA key to wrap, which must hold an ECC or RSA key
  *                  pair (see notes below).
  *
- * \note            The wrapped key must remain valid as long as the
```

Comment:
I think the bit about the validity of the wrapped key should stand out more and should remain a separate `\note`, maybe even a `\warning`.